### PR TITLE
🎉🎉🎉 Remove legacy workspace and purge MAAS of AngularJS! 🎉🎉🎉

### PR DESCRIPTION
**Pull request by Caleb-Ellis**
Huge thanks to:
@huwshimi, @Caleb-Ellis, @squidsoup, @petermakowski, @steverydz, @anthonydillon, @ClementChaumel, @bartaz, @sparkiegeek, @bethcollins92, @andesol, @sowasred2012, @hatched and @petesfrench!

## Done

- END IT

## QA steps

- Check that everything still works when you run `yarn start`

## Fixes

Fixes canonical-web-and-design/app-tribe#955

### CODE REVIEW

File: cypress/integration/docs-links/docs-links.spec.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import docsUrls from "@maas-ui/maas-ui/src/app/base/docsUrls";
+import docsUrls from "../../../src/app/base/docsUrls";
```
Comment (Caleb-Ellis) [inline cypress/integration/docs-links/docs-links.spec.ts:1]: driveby fix for docs URLs check - we can set up an import resolver in a follow up so for now it's just a relative import

**Comment by huwshimi** [comment]
Screen Shot 2022-06-10 at 3 18 13 pm

**Comment by squidsoup** [comment]
Congrats! ❤️

**Review by huwshimi — APPROVED** [review]
I'd leave a part emoji here but I think you've used them all up.

**Comment by bartaz** [comment]
image

**Comment by andesol** [comment]
Wow! :tada:

**Comment by anthonydillon** [comment]
🎉 Congratulations all involved! Thank you so much for your tireless work on this rewrite. Amazing huge improvement. Sets up the MAAS UI project for success. 

👍 The most amazing thing is that the project was perfectly on time from our planning sessions from the beginning.

**Comment by petermakowski** [comment]
🎉 🎉 🎉

**Comment by bethcollins92** [comment]
🥳🥳🥳🥳🥳🥳 Congrats!

**Comment by hatched** [comment]
party

Great work everyone!

**Review by hatched — APPROVED** [review]

**Pull request merged**